nebula_hw_interfaces/include/nebula_hw_interfaces/nebula_hw_interfaces_common/connections/udp.hpp

```diff
@@ -0,0 +1,383 @@
+// Copyright 2024 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
```

Comment:
nit: are we moving to header blocks?

Reply:
Do you mean
```cpp
#ifndef ...
#define ...
```
versus
```cpp
#pragma once
```
?

I strongly prefer pragma once for style, is there any discussion on this I have missed?

Reply:
I also prefer pragma once, but you once told me something about them nor being completely in a standard? (maybe I am misremembering)

However, in nebula there is:
`NEBULA_HesaiRosOfflineExtractBag_H`
`NEBULA_HESAI_COMMON_H`
`#pragma once`

So I though to ask about whether there is a direction that we are following from now (this is not really a blocking comment, just a question)

Reply:
Ah, thanks for the clarification! Yes, `#pragma once` is technically non-standard but in fact supported by all major compilers (for our purposes, GCC and Clang).

I think we should move to `#pragma once` for all of Nebula, thanks for pointing out that there's still old-style include guards remaining in places :pray: